plugin/output/postgres/postgres.go

```diff
@@ -43,6 +43,7 @@ const (
 	// metrics
 	discardedEventCounter  = "event_discarded"
 	duplicatedEventCounter = "event_duplicated"
+	sendErrorCounter       = "send_error"
```

Comment:
request/out _total

also let's add label status="ok|error|retry|etc", instead of separate metrics for error